karton/core/karton.py

```diff
@@ -385,3 +385,41 @@ class Karton(Consumer, Producer):
     """
     This glues together Consumer and Producer - which is the most common use case
     """
+
+    def __init__(
+        self,
+        config: Optional[Config] = None,
+        identity: Optional[str] = None,
+        backend: Optional[KartonBackend] = None,
+    ) -> None:
+        super(Karton, self).__init__(config=config, identity=identity, backend=backend)
+
+        if self.config.config.has_section("signaling"):
+            if bool(int(self.config["signaling"].get("status", 1))):
```

Comment:
Instead of doing the casts manually we can use https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.getboolean

Reply:
Good point. Let me change other occurrences of this "anti-pattern" as well.